packages/flame/lib/src/experimental/world.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:ui';
+
+import 'package:meta/meta.dart';
+
+import '../components/component.dart';
+import 'camera.dart';
+
+/// The root component for all game world elements.
+///
+/// The primary feature of this component is that it disables regular rendering,
+/// and allows itself to be rendered through a [Camera2] only. The updates
+/// proceed through the world tree normally.
+class World extends Component {
```

Comment:
I wonder if "World" is sufficiently clear that it is part of the camera system, but I can't think of anything better at the moment

Reply:
Eventually we can extend the functionality here.

For example, I was thinking that this could be a nice place to add the ability to stop or slow down the time.